hrjobcontract/tests/phpunit/fabricators/HRJobContractFabricator.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+  class HRJobContractFabricator {
+
+    private static $default = [
+      'sequential' => 1
+    ];
+
+    private static $defaultDetails = [
+      'period_start_date' => '2016-01-01'
+    ];
+
+    public static function fabricate($params, $detailsParams) {
+      if (!isset($params['contact_id'])) {
+        throw new Exception('Specify contact_id value');
+      }
+
+      $contract = civicrm_api3('HRJobContract', 'create', array_merge(self::$default, $params))['values'][0];
+
+      if (isset($detailsParams)) {
```

Comment:
Yeah initially it was like that, but then i took a look at https://github.com/civicrm/civihr/blob/staging/hrjobcontract/tests/phpunit/HRJobContractTestTrait.php#L29 and from that it seems that HRJobDetails is an integral part of a HRJobContract, unlike things like pension, or health that are "additional"